convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -164,12 +177,29 @@ def run(self):
                         " environment variable.  Please switch to 'CONVERT2RHEL_ALLOW_OLDER_VERSION'"
                         " instead."
                     )
-
+                    self.add_message(
+                        level="WARNING",
+                        id="DEPRECATED_ENVIRONMENT_VARIABLE",
+                        message=(
+                            "You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_VERSION'"
+                            " environment variable.  Please switch to 'CONVERT2RHEL_ALLOW_OLDER_VERSION'"
+                            " instead."
+                        ),
+                    )
```

Comment:
```suggestion
                    message = (
                            "You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_VERSION'"
                            " environment variable.  Please switch to 'CONVERT2RHEL_ALLOW_OLDER_VERSION'"
                            " instead."
                        )
                    logger.warning(message)
                    self.add_message(
                        level="WARNING",
                        id="DEPRECATED_ENVIRONMENT_VARIABLE",
                        message=message,
                    )
```

Remove the logger.warning above